packages/edit-site/src/components/sidebar/global-styles-v2/components/nav-link.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import { Item, NavigatorLink } from '@wordpress/components';
```

Comment:
Let's think about how we'd reconcile this with the existing navigator set of components currently in use. We can do reconciliation as a separate step, but we should be thinking about it.

Reply:
We touched this point briefly in a conversation with @griffbrad — if I remember correctly, Q had created a new set of `Navigation` components because the existing set did not satisfy his requirements. Am I remembering correctly?

Reply:
Correct. Here’s the original conversation for context on the differences: https://github.com/WordPress/gutenberg/pull/24107#issuecomment-662578255

Reply:
That's very useful. It also sheds some light on why the `Router` + `Navigation` system in this prototype don't use `react-router` directly (and instead provide a very similar, first-party implementation)